src/Symfony/Component/Routing/Tests/Generator/UrlGeneratorTest.php

```diff
@@ -720,15 +819,15 @@ public function provideLookAroundRequirementsInPath()
         yield ['/app.php/bar/a/b/bam/c/d/e', '/bar/{foo}/bam/{baz}', '(?<!^).+'];
     }
 
-    protected function getGenerator(RouteCollection $routes, array $parameters = [], $logger = null)
+    protected function getGenerator(RouteCollection $routes, array $parameters = [], $logger = null, ?string $defaultLocale = null)
```

Comment:
please remove the `?`, the default is enough

Reply:
Done.